input/pagecontent/changes.md

```diff
@@ -8,3 +8,8 @@ This change log documents the significant updates and resolutions implemented fr
 #### Changes in this version
 - New profiles: 
   - AU eRequesting Task ([FHIR-46947](https://jira.hl7.org/browse/FHIR-46947), [FHIR-46950](https://jira.hl7.org/browse/FHIR-46950), [FHIR-46956](https://jira.hl7.org/browse/FHIR-46956), [FHIR-46958](https://jira.hl7.org/browse/FHIR-46958), [FHIR-46959](https://jira.hl7.org/browse/FHIR-46959), [FHIR-47081](https://jira.hl7.org/browse/FHIR-47081), [FHIR-47082](https://jira.hl7.org/browse/FHIR-47082), [FHIR-47083](https://jira.hl7.org/browse/FHIR-47083), [FHIR-47084](https://jira.hl7.org/browse/FHIR-47084), [FHIR-47086](https://jira.hl7.org/browse/FHIR-47086), [FHIR-47102](https://jira.hl7.org/browse/FHIR-47102).
+
+- AU eRequesting ServiceRequest
+  - Made AUeRequestingServiceRequest.encounter mandatory per [FHIR-47008](https://jira.hl7.org/browse/FHIR-47008)
+
+- Renamed AU eRequesting Patient Access actor to AU eRequesting Patient and updated the description text per [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802)
```

Comment:
Don't think we need to mention descriptions, but we should mention CapabilityStatement changes; change to something like:
Renamed AU eRequesting Patient Access actor and CapabilityStatement to be AU eRequesting Patient [FHIR-46802](https://jira.hl7.org/browse/FHIR-46802)"

Reply:
This renders as a bullet point under AU eRequesting ServiceRequest changes. Please update the formatting to ensure it appears as a first-level bullet point.

Reply:
This is still unresolved.